sdk/@launchdarkly/launchdarkly_flutter_observability/example/lib/main.dart

```diff
@@ -34,10 +34,18 @@ void main() {
 
       LDSingleton.client!.start();
 
+      // Report any errors handled by flutter.
```

Comment:
So, I think we likely want this to be a step that the user takes, being as this is an assignment versus a handler registration.

Potentially we could consider a little encasulation? Like `Observe.registerFlutterError`.

Or we could change it in the options. But I like it being visible because of its implications.